app/database_etl/airtable_records_handler/airtable_records_formatter.py

```diff
@@ -62,7 +62,7 @@ def standardize_airtable_data(df: pd.DataFrame) -> pd.DataFrame:
                                 'ind_se', 'ind_se_n', 'ind_sp', 'ind_sp_n', 'jbi_1', 'jbi_2', 'jbi_3', 'jbi_4',
                                 'jbi_5', 'jbi_6', 'jbi_7', 'jbi_8', 'jbi_9', 'measure_of_age', 'number_of_females',
                                 'number_of_males', 'test_linked_uid', 'average_age',
-                                'test_not_linked_reason', 'include_in_srma']
+                                'test_not_linked_reason', 'include_in_srma', 'is_unity_aligned']
```

Comment:
Note: Christian created a lookup field for me in the estimates table for the UNITY: Criteria field. That's why it shows up as a single element list col.